library/src/main/java/tachiyomi/decoder/ImageDecoder.kt

```diff
@@ -24,7 +24,9 @@ class ImageDecoder private constructor(
   fun decode(
     region: Rect = Rect(0, 0, width, height),
     rgb565: Boolean = true,
-    sampleSize: Int = 1
+    sampleSize: Int = 1,
+    applyCMS: Boolean = false,
```

Comment:
Nitpick: expanding the abbreviation would be nice

Reply:
Done.